gcloud/datastore/connection.py

```diff
@@ -317,9 +304,9 @@ def commit(self, project, request_pb):
         request_pb.project_id = project
         try:
             return self._stub.Commit(request_pb, GRPC_TIMEOUT_SECONDS)
```

Comment:
There's a lot of credentialses running around; I get confused too (and it's why I tend to refer to them by their precise fully-qualified names).